components/admin/data-table/data-table.tsx

```diff
@@ -69,14 +69,16 @@ export function DataTable({ data, columns, onRowClicked }: DataTableProps) {
                           role="checkbox"
                           tabIndex={-1}
                           key={index}
-                          onClick={()=>onRowClicked(row, index)}
+                          onClick={() => onRowClicked(row, index)}
+                          sx={{ cursor: "pointer" }}
```

Comment:
Could the sx value be defined as a const outside of the component?
1. You avoid the duplication
2. You void unnecessary re-renders

Reply:
okay sir